docs/INCREMENTAL_UPDATES.md

```diff
@@ -143,4 +149,11 @@ OS_STATUS has 4 change(s)
 
 7 changes in Clinical Status found for this study. Outputting delta files
 36 changes in SPECIMEN found for this study. Outputting delta files
-```
\ No newline at end of file
+```
+
+## Processing outputs
++ If an `add_data/` directory exists, the script will proceed with the rest of the [whole study import workflow](WHOLE_STUDY_IMPORT.md), but will only download and process new sample or patient data found in that directory.
++ If a `delta_data/` directory exists, new corresponding `meta_*.txt` files will be generated for all associated data types, ensuring the metadata reflects the latest updates.
++ If a `delete_id_list_SAMPLE.txt` file exists, no further processing is performed on that file — it is used solely to remove samples from the portal.
+
+All additions, updates, and deletions will then be pushed to PedcBioPortal using [cBioPortal Docker Compose](https://github.com/cBioPortal/cbioportal-docker-compose).
```

Comment:
Like, kinda. Again, we can't actually access it directly. Need to meet with Charles to determine how it will actually be implemented